app/src/main/java/com/example/synerzip/recircle_android/utilities/RCWebConstants.java

```diff
@@ -1,11 +1,27 @@
 package com.example.synerzip.recircle_android.utilities;
 
+import retrofit2.http.PUT;
+
 /**
  * Created by Prajakta Patil on 7/3/17.
  * Copyright © 2016 Synerzip. All rights reserved
  */
 public class RCWebConstants {
 
```

Comment:
The key names should be in different file not in WebConstants. You can put in a file named AppConstants